Firestore/core/src/util/executor_std.cc

```diff
@@ -38,10 +56,17 @@ std::string ThreadIdToString(const std::thread::id thread_id) {
 
 }  // namespace
 
+class ExecutorStd::SharedState {
+ public:
+  // Operations scheduled for immediate execution are also put on the schedule
+  // (with due time set to `Immediate`).
+  class Schedule schedule_;
```

Comment:
`class` is necessary here because of the conflict between the `Executor::Schedule` method and the `Schedule` class, both of which are in scope here. Absent this, C++ prefers the method when resolving the unannotated name.